src/dex-ui/store/poolsSlice/utils.ts

```diff
@@ -116,6 +116,7 @@ const calculatePoolMetrics = (params: CalculatePoolMetricsParams): Pool => {
     fee: poolFee,
     totalVolumeLocked,
     past24HoursVolume,
+    tokensId: `${tokenAId}-${tokenBId}`,
```

Comment:
We needed a unique token id to differentiate and check whether the pool is already created in CreatePool Page, we can also use LPTokenId for the same.